components/rdf-ntriples.js

```diff
@@ -0,0 +1,86 @@
+// rdf-ntriples.js
+
+var _ = require('underscore');
+
+var promiseOutput = require('../src/promise-output');
+var componentFactory = require('../src/noflo-component-factory');
+
+/**
+ * Converts an RDF JS Interface Graph object into an object with the property
+ * 'tokens' containing an Array of N-Triples tokens. tokens.join('') will produce an N-Triples string.
+ */
+exports.getComponent = componentFactory({
+    description: "Converts an RDF JS Interface Graph object into an Array of N-Triples tokens",
+    icon: 'edit',
+    outPorts: promiseOutput.outPorts,
```

Comment:
We need to remember to remove this when you use the wrapper/framework - it is done for you there..